modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc

```diff
@@ -735,8 +739,12 @@ and show only active users.
 
 Here is what the full filter can look like.
 
+[source]
 ----
-&(|(objectclass=organizationalPerson))(!(userAccountControl:1.2.840.113556.1.4.803:=2))(|(|(memberof=CN=Domain Users,CN=Users,DC=dp,DC=mosreg,DC=ru)(primaryGroupID=513))))
+&(|(objectclass=organizationalPerson))
```

Comment:
Intentionally that this and the example above are the last remaining ones not transformed into a linked example file?

Reply:
Yes, because they were outside of the scope of the are that I was working on.

Reply:
ok, that´s fine.
mind to have a look on Talk?